frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -315,8 +315,8 @@ class Receive extends Component<Props, State> {
         return (
             <div class="contentWithGuide">
                 <div class="container">
-                    <Status type="warning">
-                        {paired === false && t('warning.receivePairing')}
+                    <Status type="warning" hidden={paired !== false}>
```

Comment:
Isn't it simpler to write `hidden={paired}`? It's typescript, right. So, everything is typed?

Same in send.tsx.

Reply:
initially paired is undefined because there is an async check to fetch paired. The warning should only be shown if paired is false. If paired is true or undefined it should be hidden.